soh/soh/SohMenuBar.cpp

```diff
@@ -0,0 +1,1448 @@
+#include "SohMenuBar.h"
```

Comment:
It's a shame github didn't diff this from GameMenuBar properly, can't easily see what happened here /: